test/unit/helpers/rubygems_helper_test.rb

```diff
@@ -195,4 +195,37 @@ class RubygemsHelperTest < ActionView::TestCase
       end
     end
   end
+
+  context "change_diff_link" do
+    setup do
+      @virtual_path = "rubygems.aside"
```

Comment:
doesn't seem like it is beig used

Reply:
is this for translation?

Reply:
Yes. Without it I get:

```ruby
RubygemsHelperTest#test_: change_diff_link with available version should generate a correct link to the gem versions diff. :
RuntimeError: Cannot use t(".links.review_changes") shortcut because path is not available
    app/helpers/rubygems_helper.rb:65:in `change_diff_link'
    test/unit/helpers/rubygems_helper_test.rb:224:in `block (3 levels) in <class:RubygemsHelperTest>'
    test/unit/helpers/rubygems_helper_test.rb:231:in `instance_exec'
    test/unit/helpers/rubygems_helper_test.rb:231:in `block in create_test_from_should_hash'
```

however I moved it to the relevant spec only. The yanked one can and should work without it.